runtime/etc/Makefile.exe

```diff
@@ -47,12 +47,36 @@ ifneq ($(SKIP_COMPILE_LINK),skip)
 	      $(COMP_GEN_USER_LDFLAGS) $(COMP_GEN_LFLAGS) $(LIBS) \
 	      $(CHPL_MAKE_TARGET_SYSTEM_LINK_ARGS)
 endif
+ifeq ($(COMP_GEN_DEBUG),1)
+ifeq ($(CHPL_MAKE_TARGET_PLATFORM),darwin)
+ifneq ($(CHPL_MAKE_TARGET_COMPILER),llvm)
+	dsymutil $(TMPBINNAME) -o $(TMPBINNAME).dSYM
+endif
+endif
+endif
 ifneq ($(CHPL_MAKE_LAUNCHER),none)
 	$(MAKE) -f $(CHPL_MAKE_HOME)/runtime/etc/Makefile.launcher all CHPL_MAKE_HOME=$(CHPL_MAKE_HOME) TMPBINNAME=$(TMPBINNAME) BINNAME=$(BINNAME) TMPDIRNAME=$(TMPDIRNAME) CHPL_MAKE_RUNTIME_LIB=$(CHPL_MAKE_RUNTIME_LIB) CHPL_MAKE_RUNTIME_INCL=$(CHPL_MAKE_RUNTIME_INCL) CHPL_MAKE_THIRD_PARTY=$(CHPL_MAKE_THIRD_PARTY)
 endif
 ifneq ($(TMPBINNAME),$(BINNAME))
+ifeq ($(COMP_GEN_DEBUG),1)
+ifeq ($(CHPL_MAKE_TARGET_PLATFORM),darwin)
+ifneq ($(CHPL_MAKE_TARGET_COMPILER),llvm)
+	rm -f $(BINNAME)
```

Comment:
Should we touch these things first to make sure they exist?

Reply:
I don't think that is needed. This is the same logic that was there before, `-f` meaning that `rm` won't fail if the file doesn't exist

Reply:
Ohp my bad I completely glanced over the `-f`.